llvm/lib/ObjCopy/ELF/ELFObject.cpp

```diff
@@ -2635,9 +2635,36 @@ template <class ELFT> Error ELFWriter<ELFT>::finalize() {
 }
 
 Error BinaryWriter::write() {
-  for (const SectionBase &Sec : Obj.allocSections())
+  SmallVector<const SectionBase *, 30> LoadableSections;
+  for (const SectionBase &Sec : Obj.allocSections()) {
+    if (Sec.Type != SHT_NOBITS)
+      LoadableSections.push_back(&Sec);
```

Comment:
NOBITS sections are loadable, so I don't think this variable name makes sense.

Reply:
I renamed this variable to BitsSections, maybe not the best name, but accurate.